LibreNMS/Authentication/LdapAuthorizer.php

```diff
@@ -152,6 +152,9 @@ public function getUserlist()
             }
 
             $filter = '(' . Config::get('auth_ldap_prefix') . '*)';
+            if (Config::get('auth_ldap_Userlist_filter') != null) {
+                $filter = '(' . Config::get('auth_ldap_Userlist_filter') . ')';
```

Comment:
This overrides `$filter` above, is that what you wanted to do?

Reply:
Yes this overrides $filter above only if $config['auth_ldap_Userlist_filter'] = 'your ldap filter here'; exist in the config.php

That is what I wanted to do

Reply:
Hello @laf can you approve changes in my pull request ?